resources/community/rocketchat.md

```diff
@@ -27,10 +19,51 @@ To learn more about the common channels, check out our [Chat Channel Conventions
 * Don’t know what to do? Say it out loud to our community and chances are that someone who does know will offer help.
 * Holding onto a great idea, or have a brilliant answer? Contribute it and watch the community come together to evolve it.  
 
-
 ## It's A Place To Ask Questions
 
-A good question is more likely to get an answer - and it’s likely to get a better answer more quickly, too! To that end, you'll want to include plenty of detail:
+A good question is more likely to get an answer - and it’s likely to get a better answer more quickly, too! To help you learn how to ask a good question, here are some templates to follow that might help!
+
+### I'm Having An Operational Problem
+
+Having trouble getting your deployment-config to work? Struggling to pull an image from the internal registry? Seeing a funny error message when you try to mount a PVC? Having trouble pulling from Artifactory? Is the Backup Container not working the way you expected? When you're having problems relating to your deployment on Openshift itself or with one of our operational platform tools, you'll want to make sure you include all the following information:
+
+1. **What cluster and namespace are having the problem?** Make sure you provide the `a1b2c3-env` formatted namespace name here. If you want to provide an app name too, that's great, but the community can provide much more effective help if we know where exactly to find the problem.
+1. **What tool are you using?** If you're working with a specific platform tool, make sure you let us know which one. 
+1. **What behaviour did you expect?** What was it you were trying to do? Were you building? Deploying? Mounting a PVC? Backing up a database? Let us know what it was you were trying to do and what outcome you expected.
+1. **What actually happened?** So now that we know what was *supposed* to happen, tell us all about what *did* happen. Did the build fail? Did the pod enter CrashLoopBackoff instead of starting up properly? Tell us where it all went wrong.
+1. **Include any error messages.** When something fails, chances are that there's an error message around somewhere that can tell you a little about why it failed. Make sure you include those when asking for help. Tip: put error messages into code blocks to make them easier to read in RC! And if you can't find an error message anywhere, tell us that, too (and tell us where you looked).
```

Comment:
```suggestion
1. **Include any error messages.** When something fails, chances are that there's an error message around somewhere that can tell you a little about why it failed. Make sure you include those when asking for help. Tip: put error messages into code blocks to make them easier to read in RC by wrapping them in three backticks (`` ``` ``)! And if you can't find an error message anywhere, tell us that, too (and tell us where you looked).
```
That should render properly in markdown? `` ``` ``